hackathon_site/dashboard/frontend/src/components/teamDetail/TeamInfoTable/TeamInfoTable.test.tsx

```diff
@@ -45,4 +46,41 @@ describe("Team info table", () => {
             }
         });
     });
+    it("updates checkbox on click", async () => {
+        // NOTE: this test relies on at least one checkbox being rendered, so at least one profile in mockTeam
```

Comment:
I see what you mean about this testcase lol I'll play around with it and lyk the outcome. And reminder to use team_code instead of team id now that karandeep merged his branch to develop

Reply:
okkk

Reply:
@YuYing-Liang i think i wrote all the tests before karandeep merged. they still work using team_id's though since there's no database to query against for the testing and i'm mocking the responses. do you want me to change all the tests using team_id, `api/event/teams/1`, to use team_code, `api/event/teams/ABCDE`, or can i just leave them as is?

Reply:
karandeep changed the tests to use team_code as well so to be consistent I think we should. It also avoids confusion if ppl ever look at this code in the future.